### PR TITLE
aligned question circle icon inside button

### LINKED ISSUE #78: Look into ( ? ) icon on Find Out More button
## Description
The ( ? ) icon on the Find Out More button seems slightly higher than it should be
## Plan
- [x] Check code
- [x] Fix issue

### PR DESCRIPTION
**Pull request by HedgehogHolly**
Fixes #78 

## Description

- Moves the icon to fit in with the text

🐧

### CODE REVIEW

File: src/styles/app.scss [hunk 1]
```diff
@@ -363,6 +363,14 @@ img {
 
 .button--icon {
   margin-right: 10px;
+  vertical-align: middle;
+}
+
+.button--icon.icon-question-circle-o {
+
```
Comment (katjam) [inline src/styles/app.scss]: remove the blank lines

**Review by katjam — CHANGES_REQUESTED** [review]
Good to go after removal of the extra lines.

**Comment by HedgehogHolly** [comment]
Removed the lines 👍

**Review by katjam — APPROVED** [review]
Yay! :raised_hands:

**Pull request merged**